### PR TITLE
Comments feature complete 

### PR DESCRIPTION
**Pull request by abundis28**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -47,11 +38,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Retreives textarea values.
+    String comment = request.getParameter("textarea");
+    // TODO(aabundis): Verify that a comment was written, no empty submissions.
```
Comment (gloriapark) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:45]: Nice job considering error handling!

File: portfolio/src/main/webapp/comments.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      </form>
+      <h3 class="subtitle">Past comments:</h3>
+      <!-- Container used to display the returned list children from showContent() -->
+      <div>
+        <ol id="content-container"></ol>
+      </div>
+    </div>
+  </body>
+</html>
+
```
Comment (gloriapark) [inline portfolio/src/main/webapp/comments.html]: Hm for some reason Github is saying that there's no newline at the end of the file but it looks like there is...maybe just double check that a newline does exist?

**Review by gloriapark — CHANGES_REQUESTED** [review]
LGTM! Just one random potential nit.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (abundis28) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:45]: Thanks!

File: portfolio/src/main/webapp/comments.html (hunk 2 above)
Comment (abundis28) [reply portfolio/src/main/webapp/comments.html]: There was an extra space in the last line so perhaps that was causing Github to confuse.

**Review by gloriapark — APPROVED** [review]

**Pull request merged**